src/styles/blocks/button.scss

```diff
@@ -0,0 +1,27 @@
+.button {
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #d12d35;
+  height: 56px;
+
+  font-style: normal;
+  color: #fff;
+  transition: all 0.3s;
```

Comment:
avoid using transition all, it's a bad practice
specify properties: `transition: background-color 0.3s;`